packages/docs/docs/docs/testing.md

```diff
@@ -81,9 +81,12 @@ contract('Sample', function ([_, owner]) {
 
 Use this convenient tool to write tests for your code and storage migrations before executing them in production.
 
-## Calling Initialize Functions Manually in Your Unit Tests
+## Calling Initialize Functions Manually in Your Unit Tests with Truffle 4
```

Comment:
@ajsantander are we maintaining duplicate sections for truffle 4 and 5 in the guides? If not, let's remove all references to truffle 4 here. If yes, then let's discuss how do we want to support them, given the extra effort.